packages/p2p/src/utils/date-time.js

```diff
@@ -1,17 +1,20 @@
-export const getFormattedDateString = (date_obj, is_local = false) => {
+export const getFormattedDateString = (date_obj, is_local = false, has_seconds = false) => {
     if (!(date_obj instanceof Date)) {
         throw Error('getFormattedDateString argument needs an instance of Date');
     }
 
-    const date_string = is_local ? date_obj.toString().split(' ') : date_obj.toUTCString().split(' ');
+    const date_string = !is_local ? date_obj.toString().split(' ') : date_obj.toUTCString().split(' ');
```

Comment:
Is this correct? Looks like `is_local` is now returning `UTC` strings.

Reply:
You're right, this is changed back :)